### ISSUE TITLE
Massive standard error when cutoff=NA

### ISSUE THREAD

**Issue by alexpiper** [adw96/breakaway#70]
Hi Amy,

I am attempting to use breakaway to estimate species richness from an insect microbiome sequencing experiment. When i use breakaway on the entire dataset with cutoff = NA, I am seeing massive standard errors in some samples while not in others.

![image](https://user-images.githubusercontent.com/11284720/67138761-ebb2f900-f293-11e9-9f8c-06b32f6b1068.png)

The frequency count table for the worst sample looks like this:
```
> fc
    Var1 Freq
1      1    2
2      2    1
3      3    2
4      4    1
5      5    1
6      6    2
7      9    1
8     10    1
9     14    1
10    16    1
11    18    1
12    21    1
13    22    2
14    24    1
15    25    1
16    36    1
17    51    1
18    52    1
19    80    1
20    91    1
21   774    1
22  1138    1
23  1205    1
24 24789    1

ba_tr <- breakaway(fc)
> ba_tr
Estimate of richness from method breakaway:
  Estimate is 63
 with standard error 625.51
  Confidence interval: (28, 14058)
  Cutoff:  6
```
When i change the cutoff manually to something like cutoff=3 it seems to reduce the standard error in the samples, but i get a lot of "ignoring requested cutoff; it's too low" warnings on the rest of the samples.  How would you recommend proceeding with this?

Cheers,
Alex

**Comment by adw96** [comment]
Hi Alex, thanks for using breakaway and showing me the frequency count table that is causing problems! This helps me understand the way you are using breakaway, and helps me give you the best answer I can.

A couple of things to consider
- "ignoring requested cutoff; it's too low" warnings are not bad, and they don't necessarily indicate model misspecification or cause for alarm. They just indicate that the frequency count table is "short", which means there is not a lot of data for estimating the number of missing species. I wrote in that warning to remind users not to overinterpret the total richness estimates -- essentially I want people to exercise caution and understand that there are uncertainties inherent in estimating species richness.
- If most of your samples look like the frequency count table that you recommended, it might be worth switching to the `chao_bunge` estimator instead of `breakaway`. I say this because the community you showed me is clearly dominated by one high abundance taxon. `breakaway` is better suited to more diverse communities, and is prone to overfitting in this scenario. `chao_bunge` has some limitations but does estimate fewer parameters, so might be more appropriate in this scenario. However it looks like a relatively small number of your 200+ samples (so many samples -- awesome!!!) have this problem with large standard errors, so perhaps you could stick to `breakaway` without problems (see below for why I wouldn't be bothered by this).
- Regarding large standard errors -- a large standard error on a single richness estimate indicates that we can't very precisely estimate the total species richness in that sample. However, usually folks are interested in looking at patterns of diversity across a covariate gradient -- I wrote `betta()` for doing this sort of modeling. If you plug in all of your richness estimates into `betta()`, the samples with a large standard error will be given less weight in the regression than samples with small standard errors on their richness estimates. This is the behaviour we want, because while that sample contains *some* information, the standard error is telling us that the richness estimate may not be as reliable as other samples' estimates, and we should downweight the importance of that observation relative to observations would smaller standard errors on their richness estimates.
- Regarding changing the cut-off -- this is a very reasonable thing to do when trying to get the best richness estimate possible for a specific sample. However, if you are going to use `betta()`, I recommend against tinkering with `cutoff` arguments, since with many samples, it all comes out in the wash, so to speak -- i.e., the central limit theorem kicks in and no single sample's cutoff choice will overly influence your results.

In summary, I would recommend going ahead with using `breakaway` to estimate richness on all of your samples, and then using `betta` to estimate how diversity changes across your "metadata" gradient. Again, this is the use case for most `breakaway` users, but if you're interested in something else and you have further questions, let me know and I'll try to help.

As a sanity check, you could follow the same workflow with `chao_bunge`. If the results are quite different, that may be cause for alarm/extreme caution -- and feel free to get in touch again!

Hope that helps and I'm happy to follow up if you have further questions. I just wrote a new tutorial on `betta` -- check out "Introduction to hypothesis testing for diversity" for more information.

Cheers,

Amy

**Comment by alexpiper** [comment]
Hi Amy, 

Thanks for your detailed response, that really helped clear things up for me. I proceeded as you suggested and fit the betta model to the breakaway estimates, and the results looked quite reasonable. While betta didn't find many significant associations with my metadata, this fits with my prior expectations for this system, and beta diversity is really what i am interested in for this study.

Ive enjoyed learning about the issue of alpha diversity estimation, and ended up bringing along your diversity papers to our journal club this week. It made for some very interesting discussion!

Thanks again,
Alex

**Comment by adw96** [comment]
😸
